src/components/Section/ProposalVoting/ProposalVotingSection.tsx

```diff
@@ -39,9 +40,19 @@ const ProposalVotingSection = ({
 }: Props) => {
   const t = useFormatMessage()
   let [account, accountState] = useAuthContext()
-  const [delegations] = useDelegation(account)
-  let delegate: string | null = delegations?.delegatedTo[0]?.delegate
-  let delegators: string[] = delegations?.delegatedFrom.map((delegator) => delegator.delegator)
+  let {
+      delegation,
+      delegatedVotingPower,
+      ownVotingPower
+  } = useVotingPowerInformation(account)
+  let delegate: string | null = delegation?.delegatedTo[0]?.delegate
+  let delegators: string[] = delegation?.delegatedFrom.map((delegator) => delegator.delegator)
+
+  let voteDifference:number = 0
+  if(!!delegate){
```

Comment:
Hooks shouldn't render conditionally. useVotesInformation can receive the delegate and pass the address to the other hooks conditionally to avoid api calls.